src/frontend-samples/viewer-only-2d-sample/ViewerOnly2dUI.tsx

```diff
@@ -2,125 +2,106 @@
 * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
```

Comment:
@Josh-Schifter I've made some changed to this sample (ViewerOnly2d) based on the discussion we had previously. I've modified it to not rely solely on the widget. Could you review it?